tests/test_cli.py

```diff
@@ -259,8 +277,11 @@ def test_zipfile(script_runner):
         create_zip(folder_name, tmp)
         ret = script_runner.run('geoextent','-b','-t', tmp.name)
         assert ret.success, "process should return success"
-        assert "[7.6016807556152335, 51.94881477206191, 7.647256851196289, 51.974624029877454]" in ret.stdout
-        assert "['2018-11-14', '2018-11-14']" in ret.stdout
+        result = ret.stdout
+        bboxStr = result[result.find("[") + 1:result.find("]")]
+        bboxList = [float(i) for i in bboxStr.split(',')]
+        assert bboxList == pytest.approx([7.6016807556152335, 51.94881477206191, 7.647256851196289, 51.974624029877454])
```

Comment:
Here and in several other places, the precision of the values used for testing is much higher than the one used by default in `python.approx(..)`, which is `1e-6`. For readability, I suggest to cut off all decimal places beyond the default precision of one millionth. Or am I missing something?

Reply:
I'll update the expected output to 6 decimal places.